Packs/CortexXpanse/TestPlaybooks/playbook-CortexXpanse_Test.yml

```diff
@@ -0,0 +1,961 @@
+id: CortexXpanse_Test
```

Comment:
in the test playbook no need to Delete context after each command
and please add the script `FetchFromInstance` from `Developer Tools` pack to test the fetch incidents too

Reply:
[update test playbook](https://github.com/demisto/content/pull/23592/commits/a1743913e59687fcd43e48f4b5719dafec5597f3)